.gitignore

```diff
@@ -41,10 +45,6 @@ config/config.js
 .vscode/
 yarn.lock
 
-# BUCK
-buck-out/
-\.buckd/
-*.keystore
```

Comment:
any reason we are committing these?

Reply:
As I mentioned in the previous comment, BUCK has been removed and replaced with Gradle.